src/linker/Linker/MessageContainer.cs

```diff
@@ -61,16 +65,20 @@ public static MessageContainer CreateErrorMessage (string text, int code, string
 		/// for the list of warnings and possibly add a new one</param>
 		/// /// <param name="origin">Filename or member where the warning is coming from</param>
 		/// <param name="subcategory">Optionally, further categorize this warning</param>
+		/// <param name="version">Optional warning version number</param>
 		/// <returns>New MessageContainer of 'Warning' category</returns>
-		public static MessageContainer CreateWarningMessage (LinkContext context, string text, int code, MessageOrigin origin, string subcategory = MessageSubCategory.None)
+		public static MessageContainer CreateWarningMessage (LinkContext context, string text, int code, MessageOrigin origin, string subcategory = MessageSubCategory.None, WarnVersion? version = null)
```

Comment:
Why do we default to `null` and not to `Latest`, here and also in LinkContext?

Reply:
I don't want any warnings to explicitly have version `Latest` - they should either be unversioned (always on - which is what the `null` does), or versioned with a specific number. I personally find that more clear than giving the unversioned warnings a version `Latest`, since that suggests they were recently introduced.